README.md

```diff
@@ -4,10 +4,15 @@ gopy
 [![GoDoc](https://godoc.org/github.com/go-python/gopy?status.svg)](https://godoc.org/github.com/go-python/gopy)
 [![Build Status](https://travis-ci.org/go-python/gopy.svg?branch=master)](https://travis-ci.org/go-python/gopy)
 
-`gopy` generates (and compiles) a `CPython` extension module from a `go` package.
+`gopy` generates (and compiles) a `CPython` extension module from a `go`
```

Comment:
please don't split lines like that.
ok, it's 81 characters long, but...

Reply:
Done.